torch/csrc/jit/passes/quantization/quantization_type.h

```diff
@@ -1,11 +1,14 @@
 #pragma once
+#include <iostream>
 
 namespace torch {
 namespace jit {
 
 // Quantization type (dynamic quantization, static quantization).
 // Should match the Python enum in quantize_script.py
-enum class QuantType { DYNAMIC, STATIC };
+enum QuantType : uint8_t { DYNAMIC = 0, STATIC };
```

Comment:
what's the motivation of removing enum class?

Reply:
had some compilation problem, I think we can do enum class and inherit from `uint8_t`

Reply:
this change is to make it more specific that DYNAMIC corresponds to 0 and STATIC corresponds to 1

Reply:
don't feel super strongly, but wouldn't it be good to explicitly not think of this as 0, 1, etc, and instead think of the logical value?

optional, feel free to ignore

Reply:
maybe this? https://stackoverflow.com/a/11421471

Reply:
since we cast from/to integer values I think it's better to make it more explicit about the correspondence